browser/ui/toolbar/BUILD.gn

```diff
@@ -15,38 +15,59 @@ source_set("brave_app_menu_unit_test") {
     "//testing/gtest",
   ]
 
-  sources = [
-    "//chrome/browser/ui/tabs/recent_tabs_builder_test_helper.cc",
-    "//chrome/browser/ui/tabs/recent_tabs_builder_test_helper.h",
-    "//chrome/browser/ui/tabs/recent_tabs_sub_menu_model_unittest.cc",
-    "//chrome/test/base/browser_with_test_window_test.cc",
-    "//chrome/test/base/browser_with_test_window_test.h",
-    "//chrome/test/base/menu_model_test.cc",
-    "//chrome/test/base/menu_model_test.h",
-  ]
-
   deps = [
     "//brave/app:brave_generated_resources_grit_grit",
     "//brave/app:generate_command_metadata",
-    "//build:chromeos_buildflags",
     "//chrome/app:command_ids",
+    "//chrome/browser:browser_process",
     "//chrome/browser/sync",
-    "//chrome/browser/ui:browser_navigator_params_headers",
-    "//chrome/browser/ui:ui",
     "//chrome/test:test_support",
     "//components/constrained_window:constrained_window",
-    "//components/sessions:test_support",
     "//components/sync_preferences:sync_preferences",
     "//components/sync_sessions:sync_sessions",
-    "//components/tab_groups:tab_groups",
-    "//content/test:test_support",
     "//ui/views:test_support",
   ]
   if (enable_brave_vpn) {
-    sources += [ "//brave/browser/ui/toolbar/brave_vpn_menu_model_unittest.cc" ]
+    sources = [ "//brave/browser/ui/toolbar/brave_vpn_menu_model_unittest.cc" ]
     deps += [
       "//base",
       "//brave/components/brave_vpn/common",
     ]
   }
 }
+
+source_set("brave_app_menu_browser_tests") {
+  testonly = true
+  defines = [ "HAS_OUT_OF_PROC_TEST_RUNNER" ]
+
+  if (!is_android) {
```

Comment:
don't wrap an entire target in `!is_android`. The target should be wrapped in `!is_android` instead, but in this case the file itself asserts `!is_android` so this could never be reached anyway. In a normal case you should have `!is_android` around the dep where it's used, not inside the target.

Reply:
so https://github.com/brave/brave-core/pull/26068/commits/d76f4116bb77efc4f130f847562ae6af0560d2f5#diff-02a56120c401438355c06b792cc494b41c2440cab96b107ce8fc8a49353b22f4R864 would be moved inside a `!is_android` block in the test target

Reply:
I see this was partially addressed here https://github.com/brave/brave-core/pull/26068/commits/94c16da69d111bdd79e8b863b32d8db88a79cb72#r1852907734
but we should also remove this guard

Reply:
cc: @cdesouza-chromium

Reply:
I certainly wasn't paying attention when I did this. Fixed. f07d26dcb7e